File: Data Structures & Algorithms/is-palindrome/submission-0.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        int st=0;
        
        string ans="";

        for(int i=0;i<s.length();i++){
            if(s[i]>='A' && s[i]<='Z'){
               ans+= s[i]+=32;
            }
            else if(s[i]>='a' && s[i]<='z' || s[i]>='0' && s[i]<='9'){
                ans+=s[i];
            }
        }
        int e=ans.length()-1;
        while(st<e){
            if(ans[st]!=ans[e])return false;
            else{
                st++;e--;
            }
        }
        return true;
    }
};

File: Data Structures & Algorithms/find-minimum-in-rotated-sorted-array/submission-0.java
class Solution {
    public int findMin(int[] nums) {
        int lo=0;
        int hi=nums.length;                
        int ans=0;
       
        while(lo<=hi){
            int mid=lo+(hi-lo)/2;
            
            if(nums[mid]<=nums[nums.length-1]){
                ans=nums[mid];

                hi=mid-1;
            }else{
                lo=mid+1;
            }
        }
        return ans;
    }
}
